### PR TITLE
Update chart versions

### PR DESCRIPTION
**Pull request by kahirokunn**

### CODE REVIEW

**Comment by kahirokunn** [comment]
@bryantbiggs hello 👀

**Comment by gteshomeatspoton** [comment]
@kahirokunn Thanks for making the changes. I think merging these changes will unblock a lot work.

**Comment by kahirokunn** [comment]
I hope this PR will be merged.

File: add-ons/aws-node-termination-handler/Chart.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ appVersion: "1.0"
 
 dependencies:
 - name: aws-node-termination-handler
-  version: 0.20.1
+  version: 0.20.2
```
Comment (bryantbiggs) [inline add-ons/aws-node-termination-handler/Chart.yaml:19]:
```suggestion
  version: 0.20.3
```

File: add-ons/external-dns/Chart.yaml [hunk 2]
```diff
@@ -16,5 +16,5 @@ appVersion: "1.0"
 
 dependencies:
   - name: external-dns
-    version: 6.5.6
+    version: 6.12.1
```
Comment (bryantbiggs) [inline add-ons/external-dns/Chart.yaml:19]:
```suggestion
    version: 6.13.1
```

File: add-ons/external-secrets/Chart.yaml [hunk 3]
```diff
@@ -16,5 +16,5 @@ appVersion: "1.0"
 
 dependencies:
   - name: external-secrets
-    version: 0.5.6
+    version: 0.7.0
```
Comment (bryantbiggs) [inline add-ons/external-secrets/Chart.yaml:19]:
```suggestion
    version: 0.7.2
```

File: add-ons/reloader/Chart.yaml [hunk 4]
```diff
@@ -16,5 +16,5 @@ appVersion: "1.0"
 
 dependencies:
   - name: reloader
-    version: v0.0.118
+    version: v0.0.126
```
Comment (bryantbiggs) [inline add-ons/reloader/Chart.yaml:19]:
```suggestion
    version: v0.0.129
```

**Review by bryantbiggs — APPROVED** [review]

**Pull request merged**